packages/aws-cdk-lib/aws-stepfunctions-tasks/test/glue/start-job-run.test.ts

```diff
@@ -201,3 +201,45 @@ test('Task throws if WAIT_FOR_TASK_TOKEN is supplied as service integration patt
     });
   }).toThrow(/unsupported service integration pattern/i);
 });
+
+test('Invoke glue job with WorkerType and NumberOfWorkers', () => {
+  const task = new GlueStartJobRun(stack, 'Task', {
+    glueJobName,
+    workerType: WorkerType.G_1X,
+    numberOfWorkers: 2,
+  });
+  new sfn.StateMachine(stack, 'SM', {
+    definitionBody: sfn.DefinitionBody.fromChainable(task),
+  });
+
+  expect(stack.resolve(task.toStateJson())).toEqual({
+    Type: 'Task',
+    Resource: {
+      'Fn::Join': [
+        '',
+        [
+          'arn:',
+          {
+            Ref: 'AWS::Partition',
+          },
+          ':states:::glue:startJobRun',
+        ],
+      ],
+    },
+    End: true,
+    Parameters: {
+      JobName: glueJobName,
+      WorkerType: 'G.1X',
+      NumberOfWorkers: 2,
+    },
+  });
+});
+
+test('Task throws if either workerType or num is defined', () => {
+  expect(() => {
+    new tasks.GlueStartJobRun(stack, 'GlueJob', {
+      glueJobName,
+      workerType: WorkerType.G_1X,
+    });
+  }).toThrow('');
```

Comment:
missing error msg

Reply:
I'm sorry for the careless mistake.
However, as a result of addressing the comments below, there are no more cases where an exception is thrown, so I also removed the test.